app/controllers/cards_controller.rb

```diff
@@ -43,6 +43,10 @@ def destroy
     redirect_to cards_path
   end
 
+  def test
+     @cards = Card.where("review_date < ?", Date.today)
```

Comment:
Use 2 (not 3) spaces for indentation.<br>Do not use `Date.today` without zone. Use `Time.zone.today` instead.